concourse/scripts/smoke_tests_email.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -eu
+
+DEPLOY_ENV=$1
+SYSTEM_DNS_ZONE_NAME=$2
+ALERT_EMAIL_ADDRESS=$3
+
+TO="${ALERT_EMAIL_ADDRESS}"
+FROM="${ALERT_EMAIL_ADDRESS}"
```

Comment:
this means I will receive alerts from 'me', when I set this to my own email address, which is a bit confusing...

How about setting FROM to st. like '{aws_account}-smoke-tests', e.g. 'staging-smoke-tests@digital.cabinet-office.gov.uk'

Apart from this, all else works fine...